server/unithand.cpp

```diff
@@ -3668,6 +3670,8 @@ static bool unit_bombard(struct unit *punit, struct tile *ptile,
 {
   struct player *pplayer = unit_owner(punit);
   struct city *pcity = tile_city(ptile);
+  // create QMap before loop
+  std::set<struct player *> players_to_notify;
```

Comment:
@lmoureaux confirm: this comment is wrong, right?

Reply:
Right, it's a `std::set`. And it's quite obvious that you're creating it so for me it's not needed.

Reply:
sounds good. removed